src/liger_kernel/transformers/__init__.py

```diff
@@ -10,3 +10,23 @@
     apply_liger_kernel_to_phi3,
     apply_liger_kernel_to_qwen2,
 )
+
+from .cross_entropy import LigerCrossEntropyLoss
```

Comment:
please follow the same format as above
1. use abs path
2. don't use all

Also, add a test like https://github.com/linkedin/Liger-Kernel/blob/00f2025c8783db11ecdbf6da978836665d72136c/test/transformers/test_monkey_patch.py#L16

Reply:
Sure, thanks for the review.